core/arch/aarch64/codec.h

```diff
@@ -30,8 +30,8 @@
  * DAMAGE.
  */
 
-#ifndef CODEC_H
-#define CODEC_H 1
+#ifndef CODEC_H_
```

Comment:
If we're standardizing on a style, the majority of existing headers have a leading `_` too, so let's use that (seems weird with just trailing in any case).